examples/meson.build

```diff
@@ -0,0 +1,11 @@
+runtest = files(join_paths(meson.project_source_root(), 'tests', 'runtest.sh'))
+
+tests = {
+  'INIReaderExample': { 'args': [] },
+}
+
+foreach name, properties : tests
+  test_src = 'src' in properties ? properties['src'] : 'INIReaderExample.cpp'
```

Comment:
Given it's always `INIReaderExample.cpp` here, this was confusing. What about just removing this line and hard-coding `INIReaderExample.cpp` on the next line here?